components/brave_new_tab_ui/components/default/gridSites/index.ts

```diff
@@ -55,20 +72,15 @@ export const AddSiteTile = styled('button')<AddSiteTileProps>`
   align-items: center;
   gap: 8px;
 
+  --leo-icon-size: 32px;
+  --leo-icon-color: rgba(255, 255, 255, .7);
+
   ${p => p.isDragging && css`
     visibility: hidden;
   `}
 
-  &:focus-visible, :focus {
-    gap: 4px;
-
-    ${AddSiteTileImage} {
-      margin-top: -4px;
-      width: 78px;
-      height: 78px;
-      background-clip: padding-box;
-      border: 4px solid white;
-    }
+  &:focus-visible, &:focus ${TileImageContainer} {
```

Comment:
Sorry, not sure if this is intentional but this will add the border to the `AddSiteTile` if it is `:focus-visible` but to the `TileImageContainer` if it has `:focus`.

Should this be:

```
&:focus-visible, &:focus {
 & ${TileImageContainer} {
  ...
 }
}
```

Reply:
Good catch. I'm trying to stay pretty close to the original, so I think this should be:

```
&:focus-visible, &:focus {
  ${TileImageContainer} {
    ...
  } 
}
```